lib/internal/modules/esm/formats.js

```diff
@@ -29,9 +29,9 @@ function mimeToFormat(mime) {
       /^\s*(text|application)\/javascript\s*(;\s*charset=utf-?8\s*)?$/i,
       mime,
     ) !== null
-  ) return 'module';
-  if (mime === 'application/json') return 'json';
-  if (experimentalWasmModules && mime === 'application/wasm') return 'wasm';
+  ) { return 'module'; }
+  if (mime === 'application/json') { return 'json'; }
+  if (experimentalWasmModules && mime === 'application/wasm') { return 'wasm'; }
```

Comment:
It’s in an isolated commit. I can squash together all the JSDoc commits into one and we can land this PR as two commits.

Reply:
I would prefer two PRs, it's two completely different thing, the bracket could land right away because they are very easy to review, while the JSDoc is going to take more time